include/controllers/CloudFilesController.h

```diff
@@ -71,31 +73,45 @@ class CloudFilesController : public QObject
     Q_INVOKABLE void createFolder(const QString &name);
 
 signals:
+    void updateCloudFiles(const CloudFilesUpdate &update);
+
+    void notificationCreated(const QString &notification, const bool error) const;
     void errorOccurred(const QString &errorText);
 
     void displayPathChanged(const QString &path);
     void isRootChanged(const bool isRoot);
-    void isFetchingChanged(const bool isFetching);
+    void isLoadingChanged(const bool isLoading);
```

Comment:
Using `const <primitive type>` in a function declaration is useless and takes more space.
Instead use `const <primitive type>` only in a function definition.